README.md

```diff
@@ -18,12 +18,12 @@ The latest version of the ANS schema is the 0.3.0-SNAPSHOT release, which includ
 ## Schema files
 ANS Schema files are defined with the [JSON Schema specification](https://spacetelescope.github.io/understanding-json-schema/index.html).  Schemas are defined in individual files under the [src/main/resrouces/schema/ans/_version_/](src/main/resources/schema/ans/v0_3/) directory. 
 
-Small, self-contained, attributes such as "had copyright" or "had unique id" are modeled as **traits**, which are combined together or applied directly to more complicated schema objects for the purpose of re-usability.  The "content.json" schema, for example, composes the traits "dated", "credited", "locale", "located", and "copyrighted".  The interesting schemas (story.json, image.json, and video.json) include/descend from that basic content schema and provide additional schema elements as appropriate.
+Small, self-contained, attributes such as "had copyright" or "had unique id" are modeled as **traits**, which are combined together or applied directly to more complicated schema objects for the purpose of re-usability.  The "content.json" schema, for example, composes the traits "dated", "credited", "locale", "located", and "copyrighted".  The interesting schemas (collection.json, image.json, and video.json) include/descend from that basic content schema and provide additional schema elements as appropriate.
 
 One can use a variety of third party tools to validate their content against the schema files hosted here on github publicly.
 
 ## Schema Overview
-The ARC Schema is rooted at an "ANS" node that defines an ID and Type field; the ID should be a globally unique identifier to the concrete/subclass instance of content and the Type field should describe to higher-level languages what type of content the concrete instance is (i.e. "story" or "image")
+The ARC Schema is rooted at an "ANS" node that defines an ID and Type field; the ID should be a globally unique identifier to the concrete/subclass instance of content and the Type field should describe to higher-level languages what type of content the concrete instance is (i.e. "collection" or "image")
```

Comment:
I'd roll this example back to using "story" because (a) it's a real type and (b) it's more immediately obvious to the reader of a schema overview what a "story" is compared to a "collection"